src/test/resources/hudson/plugins/jobConfigHistory/configuration-as-code.yml

```diff
@@ -0,0 +1,12 @@
+unclassified:
```

Comment:
It is a good idea to cover `true` and `false` for boolean fields.
So for example set `skipDuplicateHistory: true`.
Otherwise it might be that due to a bug in the implementation, the default value `false` for a boolean field is always used which would not be detected.

Reply:
done